backend/composer/services/cs_ingestion/helpers/statement_helper.py

```diff
@@ -50,6 +50,9 @@ def create_or_update_connectivity_statement(statement: Dict, sentence: Sentence,
             do_transition_to_invalid_with_note(connectivity_statement, error_message)
         else:
             create_invalid_note(connectivity_statement, error_message)
+    else:
+        if connectivity_statement.state != CSState.EXPORTED:
+            do_transition_to_exported(connectivity_statement)
```

Comment:
move to EXPORTED state (default) if not in EXPORTED state - if there's no validation errors. 

Reason to include this: (credit thanks - @afonsobspinto : ) )

If a statement was in INVALID state in the past; but if it has no validation errors when running it again in the future, then it should to be moved to EXPORTED state. 

To cover the fourth edge case here - https://github.com/MetaCell/sckan-composer/pull/354#pullrequestreview-2427647132